tests/commands/device/devices.spec.ts

```diff
@@ -59,4 +59,30 @@ describe('balena devices', function () {
 		// e.g. When user has a device associated with app that user is no longer a collaborator of.
 		expect(lines.some((l) => l.includes('N/a'))).to.be.true;
 	});
+
+	it('should list devices from own and collaborator apps', async () => {
+		api.scope
+			.get(
+				'/v6/device?$orderby=device_name%20asc&$select=id,uuid,device_name,status,is_online,supervisor_version,os_version&$expand=device_tag($select=tag_key,value),belongs_to__application($select=app_name,slug),is_of__device_type($select=slug),is_running__release($select=commit)',
+			)
+			.replyWithFile(200, path.join(apiResponsePath, 'devices.json'), {
+				'Content-Type': 'application/json',
+			});
+
+		const { out, err } = await runCommand('devices --json');
+		expect(err).to.be.empty;
+		const json = JSON.parse(out.join(''));
+		console.log(json);
```

Comment:
hello @bbugh This is very minor but would you mind removing this `console.log` ? I would commit amend over it but because the origin is your fork I don't have permissions.

Other than that, looks good :)

Reply:
Oh, sorry about that! That was definitely a mistake to leave in, thanks for catching it. I pushed a fix.